proposals/NNNN-DXIL-Scalarization.md

```diff
@@ -0,0 +1,90 @@
+# DXIL Scalarization
+
+* Proposal: [NNNN](NNNN-DXIL-Scalarization.md)
+* Author(s): [Farzon Lotfi](https://github.com/farzonl)
+* Sponsor: [Farzon Lotfi](https://github.com/farzonl)
+* Status: **Under Consideration**
+* Impacted Projects: Clang
```

Comment:
LLVM here?

Reply:
in our past proposals in HLSL specs we have used Clang to indicate the upstream LLVM project and DXC to indicate DXC, and Clang and DXC to indicate both. I'm Keeping that same distinction, but open to changing it.

Reply:
I think we should probably just remove the `Impacted Projects` line from the template in this repo. Nothing goes into this repo that isn't targeting LLVM/Clang.